integration-tests/testreporters/profile.go

```diff
@@ -2,6 +2,7 @@ package testreporters
 
 import (
 	"fmt"
+	"github.com/smartcontractkit/chainlink/deployment/environment/nodeclient"
```

Comment:
Is the linter complaining about this one? Or did we not configure it for this module?

Reply:
Ugh. It was - but my IDE keeps resetting it. I'll fix.

Reply:
Actually, the linter itself isn't running for me locally, so I only get the linter in the CI.